### PR TITLE
Complete website

### PR DESCRIPTION
**Pull request by benjp009**
# 👋 Micronaut Reviewer 

[Awesome books: a complete website with navigation](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md)

Today, we are working on the project first step : 

#### Part 1: Modify the Awesome books application to have:

- [x] A Navigation bar.
- [x] Three content sections:
  - [x] Books list.
  - [x] Add book form.
  - [x] Contact info.

#### Part 2: Use this wireframe as a guide:

- [x] Make sure that when you click on any of the links, the main area changes to display only one of the three content sections. (Hint: you can simply add and remove CSS classes to display or hide the content.)
- [x] To change the contents of the page, don't use any library or framework, use only plain JavaScript and CSS.
- [x] Display the current date below the navigation bar. For this requirement you should use a library - Luxon.

### Pair Programming 🎳

For this exercise, PP was required, we worked very closely with @patriciachrysy @charlesgobina  to make the best code possible for the assignment.

### Additional requirements 

- [ ] If you want to add some nice effects and transitions to the navigation, you can use CSS transitions.

Happy Review :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,22 +7,49 @@
     <title>Awesome Books</title>
```
Comment (SajjadAhmad14) [inline index.html:7]: The project is awesome and follows the wireframe. Please add all the files from [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) linter config. Moreover, please add all the unnecessary files and folders like `.vscode` to `.gitignore`.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi all, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
README is professional. :heavy_check_mark:
GitHub flow is followed. :heavy_check_mark:
## To Improve 🙏

- [x] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (benjp009) [reply index.html:7]: @SajjadAhmad14 Hello, I removed the folder `.vscode`, and we already are using the correct linter config, so I am not sure what is the error from your side ...

Comment (benjp009) [reply index.html:7]: Screenshot 2021-11-24 at 14 37 43

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi Team, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**